recipe/meta.yaml

```diff
@@ -399,10 +394,13 @@ outputs:
       ignore_run_exports:
         - __glibc
     requirements:
+      build:
+        - binutils_impl_{{ target_platform }}  {{ ctng_binutils }}
+        - binutils_impl_{{ ctng_target_platform }}  {{ ctng_binutils }}
       host:
-        - sysroot_{{ ctng_target_platform }}
+        - sysroot_{{ ctng_target_platform }} {{ conda_glibc_ver }}
       run_constrained:
-        - libgfortran-ng {{ ctng_gcc }} *_{{ build_num | default(0) }}
```

Comment:
This was here to avoid mixing the old and new packages after the gfortran rename. I guess we don't need it anymore due to the version bump.